functions/src/index.ts

```diff
@@ -17,7 +17,10 @@ app.get("/", (req, res) => {
   res.status(200).send("Hey there!");
   res.end();
 });
+app.get("/user/:user_id", getUser);
```

Comment:
Have you ever used the [express.Router class](https://expressjs.com/en/guide/routing.html)? It lets you make modular route handlers that you can abstract away into separate route files for each base endpoint url.